app/services/foreman/plugin.rb

```diff
@@ -351,6 +351,11 @@ def parameter_filters(klass)
       @parameter_filters.fetch(klass.is_a?(Class) ? klass.name : klass, [])
     end
 
+    # Add concerns to a host
+    def extend_host(concern_module)
```

Comment:
Bad idea, what does this give the plugin developer? They can just rely on Rails for that...

`https://github.com/theforeman/foreman_ansible/blob/master/lib/foreman_ansible/engine.rb#L51`